app/src/main/java/com/example/bookwormadventuresdeluxe2/BorrowDetailViewFragment.java

```diff
@@ -83,11 +88,15 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                 this.btn1.setText(getString(R.string.set_location));
                 this.btn2.setText(getString(R.string.return_book));
 
-                //TODO: get pickup location from book
-//        this.bookDetailView.findViewById(R.id.borrow_exchange).setVisibility(View.VISIBLE);
+                if(this.selectedBook.getPickUpAddress().equals("")) {
+                    this.btn2.setBackgroundTintList(getResources().getColorStateList(R.color.tempPhotoBackground));
+                    this.btn2.setTextColor(getResources().getColorStateList(R.color.colorPrimary));
+                } else {
+                    this.btn2.setOnClickListener(this::btnReturnBook);
+//                    this.bookDetailView.findViewById(R.id.borrow_exchange).setVisibility(View.VISIBLE);
```

Comment:
should this be commented out?

Reply:
this was to show the selected location string to the user. I decided to hide this at least for now since it is just a coordinate and not a nice location

Reply:
we can make the setLocationActivity return a street address later if it would look nice.